gitlens/gitlens-features.md

```diff
@@ -356,36 +380,60 @@ Minimap Overview:
 
 ***
 
-## Focus View ✨
+## Launchpad `PRO`
 
-The Focus View allows you to stream line your workflow by providing a summarized list of your pull requests and your issues. To open the Focus View, open the command palette (`command/ctrl + shift + P`) and search `GitLens+: Show Focus view`.
+The Launchpad organizes pull requests by status allowing you to identify bottlenecks and take actions on them. This offers the ability to organize and quickly take actions on pull requests and issues with priority.
```

Comment:
```suggestion
The Launchpad organizes pull requests by status allowing you to identify bottlenecks and take actions on them. This offers the ability to organize and quickly take actions on pull requests with priority.
```

Suggest removing reference to issues here. Even though the old editor version of Launchpad supports issues, the new one being emphasized here doesn't yet.